src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -454,6 +454,31 @@ def closeBlockContainer(self, name, scope='cms'):
             self.logger.error("Exception closing container/block: %s. Error: %s", name, str(ex))
         return response
 
+    def moveReplicationRule(self, ruleId, rseExpression, issuer):
+        """
+        Perform move operation for provided rule id and rse expression
+        :param ruleId: rule id
+        :param rseExpression: rse expression
+        :param issuer: rucio quota account
```

Comment:
To keep things consistent, I would suggest to rename `issuer` to either `acct` or `account`

Reply:
ok